packages/@glimmer/local-debug-flags/index.ts

```diff
@@ -1,5 +1,10 @@
+let location = typeof window !== undefined && window.location;
+let DEBUG = false;
+if (location && /[?&]glimmer_logging/.test(window.location.search) {
```

Comment:
strange. Anyways, thanks for fixing! Sorry about the extra work.  (hopefully the now faster tests will offset)